shareplugin/translations/sailfish-qrshare-pt_PT.ts

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="pt_PT">
+<context>
+    <name></name>
+    <message id="qrshare-display_name">
+        <source>QR Share</source>
+        <extracomment>Plugin display name</extracomment>
+        <translation>Gerar código QR</translation>
+    </message>
+    <message id="qrshare-menu-save_to_gallery">
+        <source>Save to Gallery</source>
+        <extracomment>Pulley menu item</extracomment>
+        <translation>Guardar na galeria</translation>
+    </message>
+    <message id="qrshare-placeholder-no_text">
+        <source>Nothing to share.</source>
+        <extracomment>Placeholder text</extracomment>
+        <translation>Nada para partilhar</translation>
+    </message>
+    <message id="qrshare-placeholder-qrcode-too_long">
+        <source>Text is too long for QR code.</source>
+        <extracomment>Placeholder text</extracomment>
+        <translation>Texto demasiado longo para código QR.</translation>
+    </message>
+    <message id="qrshare-placeholder-aztec-too_long">
+        <source>Text is too long for Aztec code.</source>
+        <extracomment>Placeholder text</extracomment>
+        <translation type="unfinished">Texto demasiado longo para código Aztec.</translation>
```

Comment:
I leave `"unfinished"` for the strings that haven't been verified by a native speaker (or not translated at all). This `"unfinished"` should't be here but I can remove it myself, no problem.

Reply:
So, who is your native speaker for pt_PT? me?? ;)

Reply:
I was assuming so!